ui/app/templates/components/transit-key-action/hmac.hbs

```diff
@@ -1,4 +1,4 @@
-<form {{action 'doSubmit' (hash input=input algorithm=algorithm key_version=key_version) on="submit"}}>
+<form {{action 'doSubmit' (hash input=input hash_algorithm=hash_algorithm key_version=key_version) on="submit"}}>
```

Comment:
It's strange that these are snake cased. Is that to avoid conversion before sending a hash over to the server?

Reply:
Yeah exactly - we pass them directly to the adapter this way, there's no serialization done. Moving this to the model-backed approach would get rid of this too. I also dislike how the template is responsible for passing the proper attributes, and it's something I've tripped over before.